geniza/corpus/views.py

```diff
@@ -84,17 +92,50 @@ class DocumentDetailView(DetailView):
 
     context_object_name = "document"
 
+    def page_title(self):
+        return self.get_object().title
+
+    def page_description(self):
+        return Truncator(self.get_object().description).words(20)
+
     def get_queryset(self, *args, **kwargs):
         """Don't show document if it isn't public"""
         queryset = super().get_queryset(*args, **kwargs)
         return queryset.filter(status=Document.PUBLIC)
 
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        context_data.update(
+            {
+                "page_title": self.page_title(),
+                "page_description": self.page_description(),
```

Comment:
Great to have these conventions established early — should every page have these?

Reply:
I think so, yeah!